execution-engine/engine-tests/src/test/regression/mod.rs

```diff
@@ -13,8 +13,6 @@ mod ee_572;
 mod ee_584;
 #[cfg(feature = "use-system-contracts")]
 mod ee_597;
-#[cfg(feature = "use-system-contracts")]
-mod ee_598;
```

Comment:
Why..?
Can we add step execution?

Reply:
This is also checked in [here](https://github.com/hdac-io/CasperLabs/blob/f64b7c414f53aedea6bae13d4dc3b1eb599d33b8/execution-engine/engine-tests/src/test/hdac_system_contracts/proof_of_stake/delegation.rs#L508) :)

Reply:
Wouldn't it be better to keep the code and separate it from our features?

I understood that we left all of Casper's test code, but I'm not sure what the difference is.

Reply:
Every test suites except for `engine-tests/src/system-contracts` runs based on our system contract(hdac_system). So ee_598 has also tested our system contracts(pop). The motivation why I delete is that ee_598_regression.wasm includes bond and unbond in a single function call.
So, to apply step functionality I had to break down ee_598_regression, but since we already checked the same logic in our test code, I made a decision to remove it rather than modifying it. Only test suites in `engine-tests/src/system-contracts` test the features of `system-contracts/(pos, mint)` through `create_casper_genesis_config()` and `DEFAULT_CASPER_GENESIS_CONFIG`.

If you feel something unclear, please let me know. :)